cluster/manager.go

```diff
@@ -46,26 +47,62 @@ var (
 	unmarshalErrMergeRemoteState = stats.NewCounter32("cluster.decode_err.merge_remote_state")
 )
 
-type ClusterManager struct {
+type ClusterManager interface {
```

Comment:
this goes against go's "no stuttering" naming convention. can just call this Manager since it's in package cluster

Reply:
there is a pacakge var called "Manager" already.